src/com/epam/javacourse/homework2/Fibonacci.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.javacourse.homework2;
+
+public class Fibonacci {
+    public void getFibonacciRow(int n) {
+        int first = 0;
+        int second = 1;
+        System.out.println("Fibonacci Series:");
+
+        for (int i = 1; i <= n; ++i) {
+            System.out.printf(first + " ");
```

Comment:
Better to be replaced with `System.out.print`

Reply:
fixed.